app/models.py

```diff
@@ -94,3 +95,62 @@ def facet_plural(facet: str):
         facet_plural = facet
 
     return facet_plural
+
+
+class ElementsItem(BaseModel):
```

Comment:
should be name "TextElementItem" (if you would have an image, it would be different)